tests/compile-fail/non_expressive_names2.rs

```diff
@@ -0,0 +1,14 @@
+#![feature(plugin)]
```

Comment:
stuff wasn't working and compiletest doesn't produce nice error messages. I'll merge it back in the normal file